Podfile

```diff
@@ -12,8 +12,9 @@ install! 'cocoapods',
 
 def common_pods
   pod 'Adyen', '3.7.0'
+  pod 'KarhooSDK', :git => 'git@github.com:karhoo/karhoo-ios-sdk.git', :commit => 'e00e9df888ffe9fa75b29e6c4ecdeca802254464'
```

Comment:
I believe this should not be merged into `develop`.

Reply:
It would be better to point to a branch instead of commit

Reply:
This is temporary solution, if changes appear to be correct then we should release new version for NSDK and point do develop branch again

Reply:
This changes should be aplied to new NSDK release:
https://github.com/karhoo/karhoo-ios-sdk/commit/13018d4e940b85d3149b35b8ab9f3e5c162ec073